airflow/providers/trino/hooks/trino.py

```diff
@@ -145,6 +145,7 @@ def get_conn(self) -> Connection:
             verify=_boolify(extra.get("verify", True)),
             session_properties=extra.get("session_properties") or None,
             client_tags=extra.get("client_tags") or None,
+            timezone=extra.get("timezone") or None,
```

Comment:
Should be also added into the [Documentation](https://airflow.apache.org/docs/apache-airflow-providers-trino/stable/connections.html):
- https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-trino/connections.rst